src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -112,6 +115,40 @@ private CallbackValidations() {
             : invalid("Exception case has no Id");
     }
 
+    @Nonnull
+    public static Validation<Seq<String>, CaseDetails> hasValidDetailsForAttachingToCase(
+        boolean useSearchCaseReference,
+        CaseDetails theCase
+    ) {
+        Validation<String, String> caseReferenceTypeValidation = useSearchCaseReference
+            ? hasSearchCaseReferenceType(theCase)
+            : valid(CCD_CASE_REFERENCE);
+        Validation<String, String> caseReferenceValidation = useSearchCaseReference
+            ? hasSearchCaseReference(theCase)
+            : hasAttachToCaseReference(theCase);
+        Validation<String, String> jurisdictionValidation = hasJurisdiction(theCase);
+        Validation<String, String> serviceNameInCaseTypeIdValidation = hasServiceNameInCaseTypeId(theCase);
+        Validation<String, Long> idValidation = hasAnId(theCase);
```

Comment:
has service case id or exception case id ?

Reply:
hasAnId is an existing method, should not be renamed in this PR. It is checking ID of the exception record